Description: Use correct predicate before calling get_colr_glyph_paint()
 This ensures Chromium doesn't crash when the experimental COLRv1 API is disabled.
Origin: https://gitlab.freedesktop.org/freetype/freetype/-/commit/16f311d72582c117796a23e22074fe9624760ee1
Author: Hugh McMaster <hugh.mcmaster@outlook.com>
Forwarded: not-needed
Last-Update: 2024-02-24

--- a/src/base/ftobjs.c
+++ b/src/base/ftobjs.c
@@ -5746,7 +5746,7 @@
     ttface = (TT_Face)face;
     sfnt   = (SFNT_Service)ttface->sfnt;
 
-    if ( sfnt->get_colr_layer )
+    if ( sfnt->get_colr_glyph_paint )
       return sfnt->get_colr_glyph_paint( ttface,
                                          base_glyph,
                                          root_transform,
